src/uct/ib/rc/base/rc_ep.c

```diff
@@ -208,22 +208,31 @@ uct_rc_ep_send_op_completed_iov(uct_rc_iface_send_op_t *op)
 }
 
 static UCS_F_ALWAYS_INLINE void
-uct_rc_op_release_get_bcopy(uct_rc_iface_send_op_t *op)
+uct_rc_op_release_get_bcopy(uct_rc_iface_send_op_t *op, int release)
 {
     uct_rc_iface_send_desc_t *desc = ucs_derived_of(op, uct_rc_iface_send_desc_t);
     uct_rc_iface_t          *iface = ucs_container_of(ucs_mpool_obj_owner(desc),
                                                       uct_rc_iface_t, tx.mp);
-
-    iface->tx.reads_available += op->length;
+    if (release) {
+        iface->tx.reads_available += op->length;
```

Comment:
can take `iface` from the `op` like it is done in `uct_rc_op_release_get_zcopy()`?
so, if yes, then could remove `desc` and `iface` variables

Reply:
no, iface is in union in ops, and is not always available

Reply:
should we fix the function below then?

Reply:
`ops->iface` is available for zcopy completions, but not for bcopy.

Reply:
I see
thanks